src/fides/api/ops/api/v1/endpoints/privacy_notice_endpoints.py

```diff
@@ -161,11 +162,17 @@ def get_privacy_notice(
     *,
     privacy_notice_id: str,
     db: Session = Depends(deps.get_db),
+    request: Request,
 ) -> PrivacyNotice:
     """
     Return a single PrivacyNotice
     """
-    return get_privacy_notice_or_error(db, privacy_notice_id)  # type: ignore[return-value]
+    should_unescape = request.headers.get("unescape-safestr")
+    notice = get_privacy_notice_or_error(db, privacy_notice_id)
+    if should_unescape:
+        print("orig", notice.name)
```

Comment:
I can't get onboard with this abbreviation 😂 

```suggestion
        print(f"Original Value: {notice.name}")
```

Reply:
haha that was just for debugging! I'll remove it 👍